src/wallet/load.cpp

```diff
@@ -62,21 +62,19 @@ bool VerifyWallets(interfaces::Chain& chain)
     std::set<fs::path> wallet_paths;
 
     for (const auto& wallet_file : gArgs.GetArgs("-wallet")) {
-        const fs::path path = fs::absolute(wallet_file, GetWalletDir());
-
-        if (!wallet_paths.insert(path).second) {
-            chain.initError(strprintf(_("Error loading wallet %s. Duplicate -wallet filename specified."), wallet_file));
-            return false;
-        }
-
         DatabaseOptions options;
         DatabaseStatus status;
         options.verify = true;
         bilingual_str error_string;
-        if (!MakeWalletDatabase(wallet_file, options, status, error_string)) {
+        std::unique_ptr<WalletDatabase> database = MakeWalletDatabase(wallet_file, options, status, error_string);
+        if (!database) {
             chain.initError(error_string);
             return false;
         }
+        if (!wallet_paths.insert(database->Filename()).second) {
```

Comment:
Since `MakeWalletDatabase` will have a duplicate open check, this error will never be hit. Instead the user will get a less useful error about `Another instance of bitcoin may be using it`.

Reply:
What PR changes it? SQLite?